lib/modules/dosomething/dosomething_api/resources/signup_resource.inc

```diff
@@ -189,7 +191,120 @@ function _signup_resource_index($user, $users, $campaigns, $count, $page, $compe
     'runs' => $runs,
   ];
 
-  return (new SignupTransformer)->index($parameters);
+  if (variable_get('rogue_collection', FALSE)) {
+    if ($user) {
+      $northstar_user_id = dosomething_user_get_northstar_id($user);
```

Comment:
It looks like `$user` may sometimes be an array or a string with comma-separated values (not sure which). Can you make sure this still works if in that case?